wfsim/core/s2.py

```diff
@@ -221,7 +231,9 @@ def get_electron_yield(n_electron, xy_int, z_int, config, resource):
         # Absorb electrons during the drift
         electron_lifetime_correction = np.exp(- 1 * drift_time_mean /
                                               config['electron_lifetime_liquid'])
-        cy = config['electron_extraction_yield'] * electron_lifetime_correction
+        cy = config['g2_mean']*resource.s2_correction_map(xy_int)*\
```

Comment:
The S2 correction map and the gain are already included in the S2 light yield, isn't including them here as well equivalent to apply the correction twice?

Reply:
Hi Francesco, so there's sort of two different uses in the electron vs light yield. In the light yield, it either uses the s2_correction_map (if using the old method) or the se_gain_map (using the newer update). This gives the number of photons per electron. For the electron yield, it uses the ratio of g2/SE_gain to get extraction efficiency, so this correction is to get the number of electrons which are extracted. The light yield gets the photons per extracted electrons. I hope this makes sense.